mw-webapp/src/component/editableTextarea/editableTextarea.tsx

```diff
@@ -93,8 +100,9 @@ export const EditableTextarea = (props: EditableTextareaProps) => {
       onBlur={handleChangeFinish}
       onKeyDown={handleCtrlEnter}
       className={clsx(styles.editableTextarea, props.className)}
+      // className="grow-wrap"
     >
-      {isEditing ? renderTextarea() : renderSpan(text)}
+      {isEditing ? renderTextarea() : <span style={{ whiteSpace: 'pre-wrap' }} >{text}</span>}
```

Comment:
son't use inline styles

Reply:
At that time I didn’t find another solution, but everything is fine ;)